dashboard/test/models/pd/workshop_test.rb

```diff
@@ -44,6 +44,28 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     assert_equal workshops.first, @workshop
   end
 
+  test 'facilitated_or_organized_by' do
```

Comment:
Thanks for this - I tested this at the controller level but not with this level of detail